libs/ui/src/components/NeoModal/NeoModalBody.vue

```diff
@@ -0,0 +1,167 @@
+<template>
+  <div class="modal-width">
+    <header
+      class="px-6 is-flex is-justify-content-space-between border-bottom is-align-items-center"
+      :class="{
+        'py-5': !loading,
+        'py-4': loading,
```

Comment:
why different size between?

Reply:
then reduce size of skeleton? it shouldn't affect content spacing/size

Reply:
I mean i'm ok , but then it wont match the design exactly

Reply:
increasing the height without reducing the top box padding makes the height of the top box change when it stops loading, 

with these two padding, the top box height does not change. when it goes from loading to not loading

Reply:
How about adding `line-height` to the text?